imagick/resize.go

```diff
@@ -0,0 +1,141 @@
+package imagick
+
+import (
+	"github.com/journeymidnight/pipa/helper"
+	"github.com/journeymidnight/pipa/library"
+	"gopkg.in/gographics/imagick.v3/imagick"
+	"math"
+)
+
+const Method = imagick.FILTER_LANCZOS
```

Comment:
If it is zoomed in, please use "MITCHELL" algorithm, see related analysis for details